### ISSUE TITLE
keyboard template not working

### ISSUE THREAD

**Issue by taviso** [taviso/wpunix#27]
Help > Template doesn't seem to work, it's supposed to show a keyboard template.

**Comment by sieger82** [comment]
I have the same issue, any idea how to fix it?

**Comment by taviso** [comment]
No, you can press <kbd>Shift</kbd>+<kbd>F1</kbd>,<kbd>4</kbd>,<kbd>7</kbd> and it shows bindings though.

I think the template should be like ASCII art, I'm not sure why it's not working... I'll have to step through the code!

**Comment by emendelson** [comment]
It probably looks something like this (from WPDOS 5.1+):

<img width="913" height="545" alt="Image" src="https://github.com/user-attachments/assets/775451b9-0fc5-4e2c-b57f-0af42ed4dcce" />

**Comment by sieger82** [comment]
tnx for the screenshot

**Comment by sieger82** [comment]
I found a fix for this. As it turns out, the template flag needs to be set in the xterm.trs file. To do this, I used the WordPerfect Terminal Definition program:
```
cd /opt/wp80/shlib10
sudo ../shbin10/wpterm
```
as described in the admin manuals linked at the bottom of this page: https://mendelson.org/wpdos/unix.html
it is important to run the `wpterm` utility from inside `/opt/wp80/shlib10`.

now highlight `xterm` or `xterm-256color` and press `7` to 'look/edit'. then press `7` again for 'Template flag'. It will be set to 'F. Potpourri-Style Function Layout'. Press `a` and then `y` to select the 'PC-Style function layout'.
then `0`,`0`. `y` to save, `y` to overwrite, `0` to exit.

Now start `wp` again and the template menu option (and also F3,F3) will work.

**Comment by taviso** [comment]
Huh, so that's what that flag does. Thanks for tracking it down!
